addons/common/fnc_headDir.sqf

```diff
@@ -75,8 +75,8 @@ if (_unit != call CBA_fnc_currentUnit) then {
 
 private _diff = -_azimuth;
 
-if (!isNull _offsetObject) then {
-    ADD(_diff,_unit getDir ([_offsetObject] call CBA_fnc_getPos));
+if !(_offset isEqualTo "") then {

```

Comment:
If you supply any other string than an empty string, it will return true:
`!("base_marker" isEqualTo "")` will return true.